doc/changes/latest.rst

```diff
@@ -101,3 +101,5 @@ Changes
 - :bdg-dark:`Code` Reorder condition in internal call of :func:`nilearn.image.resample_img` to skip checking of array values if interpolation is ``nearest`` (:gh:`4571` by `Jason Kai`_).
 
 - :bdg-dark:`Code` Remove redundant sorting of ``np.unique(data)`` in internal call of :func:`nilearn.image.resample_img` when checking array values (:gh:`4571` by `Jason Kai`_).
+
+- :bdg-primary:`Doc` Add missing default values to the docstrings in 'nilearn/glm'(:gh:`4656` by `Anupriya Kumari`_).
```

Comment:
Note that you will have to add yourself to the citation.cff otherwise the automated build of the doc will fail.

Reply:
Oh right! Slipped off my mind, thanks.